README.md

```diff
@@ -981,6 +981,7 @@ Table of Contents
    * [codesandbox.io](https://codesandbox.io/) — Online Playground for React, Vue, Angular, Preact and more.
    * [Eclipse Che](https://www.eclipse.org/che/) - Web based and Kubernetes-Native IDE for Developer Teams with multi-language support. Open Source and community driven. An hosted version is available at [che.openshift.io](https://che.openshift.io/)
    * [fakejson.com](https://fakejson.com/) — FakeJSON helps you quickly generate fake data using its API. Make an API request describing what you want and how you want it. The API returns it all in JSON. Speed up the go to market process for ideas and fake it till you make it.
+   * [GitHub Codespaces](https://github.com/features/codespaces) - a cloud-hosted, containerized, and customizable VS Code environment in your browser. Early access available.
```

Comment:
If not already there please add a mention to the existing github entry instead, I'd prefer not to add another entry for them